src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -103,7 +103,27 @@ object Lorre extends App with TezosErrors with LazyLogging with LorreAppConfig {
     tezosConf.network
   )
 
-  try {mainLoop(0)} finally {shutdown()}
+  /**
+    * Tries to fetch blocks head to verify if connection with Tezos node was successfully established
+    */
+  @tailrec
+  def checkConnection(): Unit = {
+    Try {
+      Await.result(tezosNodeOperator.getBlockHead(tezosConf.network), Duration.Inf)
```

Comment:
I would put a limited duration as a timeout, as we actually want to check repeatedly if no timely response comes back

Reply:
it also times out with `Duratinion.Inf`, but the question is, do you have any specific value in mind?

Reply:
after consulting with @ivanopagano added 10 seconds timeout in the Lorre config